CMakeLists.txt

```diff
@@ -35,9 +35,13 @@ set(${PROJECT_NAME}_SRCS
   src/multi_library_class_loader.cpp
 )
 add_library(${PROJECT_NAME} SHARED ${${PROJECT_NAME}_SRCS})
-target_include_directories(${PROJECT_NAME}
-  PUBLIC include ${console_bridge_INCLUDE_DIRS} ${Poco_INCLUDE_DIRS})
-target_link_libraries(${PROJECT_NAME} ${console_bridge_LIBRARIES} ${Poco_LIBRARIES})
+if(ament_cmake_FOUND)
```

Comment:
Is this conditional correct? This reads to me like `ament_target_dependencies` would be used if ament_cmake is not found.

Reply:
Wrong if/else order. Corrected